ambuda/seed/texts/gretil.py

```diff
@@ -115,6 +115,8 @@ def run():
         add_document(engine, spec)
     log("Done.")
 
+    return True
```

Comment:
remove this line?

Reply:
The [`init_database()`](https://github.com/kvchitrapu/ambuda/blob/main/scripts/setup_database.py#L53) calls `run()` in lookup, gretil, and dcs modules. The returned result is a boolean and an error message is logged in case of an error. This pattern helps us reliably integrate new modules. These sources are generally up, but best to log errors and fail than proceed at this time.